### ISSUE TITLE
Add section list to default JSON exports

### ISSUE THREAD

**Issue by jaburgoyne** [Amsterdam-Music-Lab/MUSCLE#537]
We've expanded our JSON exports to include good information about sessions and participants as separate trees. Given this structure, it would be convenient also to include a tree listing information about *sections*. Although this will duplicate the playlist CSV file in many cases, for debugging and future analysis after an experiment may have been taken down, it would be useful to have a record of what metadata the MUSCLE infrastructure had attached to a particular section_id for a particular experiment.

In short, an ideal JSON export should dump the complete data structure for sessions, participants, *and* sections.

**Comment by BeritJanssen** [comment]
Before we implement a lot of stuff from scratch - perhaps the `dumpdata` command that ships with Django is also sufficient here? BTW there is no direct link between a section and a participant - they are both linked to results, so in that sense there is a connection. But we don't keep participant-wise play counts or anything. Even though this can be referred back to from the results in an experiment. So I'm not so sure anything is not in the json currently that would be needed for analysis, nor do I think any data is lost if the experiment is taken down. Recently we had some data loss, but this was because of a heavy restructuring of the database, which is unlikely to happen again. In fact, with backups we can always go back to earlier stages of the database (so that data pre-April is also not lost for good, just takes a bit of fiddling to load into a database).

**Comment by BeritJanssen** [comment]
Perhaps we can overhaul the json export: throw out the custom json export code, and instead use Django's built in serializers. So a click on the 'json' button would trigger a `./manage.py dumpdata session.Session --pks {list_of_sessions_for_requested_experiment}`, and likewise for the associated participants and sections.
[Django documentation](https://docs.djangoproject.com/en/4.2/ref/django-admin/#dumpdata)
As discussed, this could be delivered to the user as separate json files (zipped).

**Comment by BeritJanssen** [comment]
Also, have a look at standard Django serializers and the [Django-Rest framework](https://www.django-rest-framework.org/api-guide/serializers/).
